book-worm-2010/book-worm-fe
# Feature/refactor accessibility animations

### What does this PR do? 
>
Fixes errors in console, including adding features for accessibility and routing from FinishBookForm to Celebration.  
Accessibility score was improved to 100 on Lighthouse by changing contrast color ratios and adding alt tags to all images.  Clean up and refactor was done.  Styling was refactored to add a media query to the home page to scale the display.  

### Is this a feature or Fix/Refactor?  

Feature, Fix, and Refactor 

### Problems Encountered & Resulting Solutions  
>
GSAP displays warnings in the console that don't seem to match up with what is being rendered (i.e. `GSAP target not found`).  Upon researching the GSAP docs and FAQs, it seems to be nothing to worry about but can be reconfigured in .config settings later on. 

I've noticed that the celebration animation doesn't always play on redirect to the link but I'm not sure if that's because of the console open or some other issue to be determined.  For now, the component displays and works.


### Where should the reviewer start?
>
Celebration.js, Home.js, FinishBookForm.js and App.js for routing the Celebration component from the Finish Book form.  
SCSS files were also updated.
Component files were updated to import GSAP.

### How should this be tested?
>
Run `npm start` in terminal when in the project directory and go to localhost:3000 in your browser URL to see the display in action.  Login with your google account and click to add a book.  The added book should show up on the path display on the home page.  Click on the book to see book details to add a bookmark.  The bookmark page shows a button to "finish book"-- after clicking, fill out the form and hit submit.  Upon submitting, you should be taken to a celebration page to view the animation.

On the home page, scale in the browser to narrow the window to see the navigation bar scale in size.

<img width="502" alt="Screen Shot 2021-04-19 at 8 54 29 PM" src="https://user-images.githubusercontent.com/67291333/115331871-f9309400-a153-11eb-9833-2c73552fc0db.png">

### Steps To Take
>
Once the other pages are done with styling, we need to make sure the page is still hitting a high accessibility score and media queries are added where needed.
